### PR TITLE
Added Refiners to ResultTable interface (SharePoint Search)

### PR DESCRIPTION
**Pull request by nickvanhooydonk**
| Q               | A
| --------------- | ---
| Bug fix?        | [ ]
| New feature?    | [ x ]
| New sample?      | [ ]
| Related issues?  | 

#### What's in this Pull Request?

Added Refiners to ResultTable interface (SharePoint search).
With Refiners added in this interface we can use these refiners as a result from a SharePoint search query.

### CODE REVIEW

File: debug/debug.ts [hunk 1]
```diff
@@ -32,8 +32,11 @@ Logger.activeLogLevel = LogLevel.Info;
 // add your debugging imports here and prior to submitting a PR git checkout debug/debug.ts
 // will allow you to keep all your debugging files locally
 // comment out the example
-import { Example } from "./example";
-Example();
+// import { Example } from "./example";
+// Example();
+
+import { TestCase } from "./debug-search";
```
Comment (patrick-rodgers) [inline debug/debug.ts]: Can you revert the debug.ts file so your local changes aren't included?

`git checkout debug/debug.ts` should do it.

**Comment by nickvanhooydonk** [comment]
@patrick-rodgers I was messing a little bit with the Git remote repository, so eventually I had to revert a commit. So your link to the changes is not visible anymore because that commit is gone.

But there is one file changed now and the debug/debug.ts is removed as requested.
So I think we are good to go!

If I can help you with something else please let me know!
Thanks!

**Comment by patrick-rodgers** [comment]
Thanks for your contribution!

**Comment by patrick-rodgers** [comment]
Oh, could you provide your name how you'd like it to appear, otherwise I can list you as nickvanhooydonk in the authors file? Thanks!

**Comment by nickvanhooydonk** [comment]
Nick van Hooydonk or nickvanhooydonk. Both fine 👍

**Pull request merged**